src/cdsetool/download.py

```diff
@@ -44,7 +44,9 @@ def download_feature(
         log.debug(f"Bad URL ('{url}') or title ('{title}')")
         return None
 
-    filename = title.replace(".SAFE", ".zip")
+    filename = title.replace(".SAFE", ".SAFE.zip")
```

Comment:
Does it work if you make this line `filename += ".zip"` instead?

Reply:
Sure thing. However, `filename` is a new variable, which is created through `title`. 
So it would have to be `filename = title + ".zip"`

I have made these changes so not to change the coding style. But it can be done as per your proposal, no problem.

Reply:
Sorry, I was in the middle of something else when making my comment. You're right, `filename = title + ".zip"` is what I had in mind, and should have written.

The reason I'm suggesting to just append the `".zip"` instead of having an exhaustive list of collections is that they keep making more data available from Copernicus over time so the list needs to be kept up to date. I know they recently added Sentinel-3 OLCI products, and apparently they have been providing Landsat data for some time already.

Reply:
Sure, no problem. If this the only comment, I will proceed with the change.
For sure it's much cleaner and lighter. Do you have in mind any cases which this append would not be sufficient?

Reply:
Well, it's hardly any guarantee, but I can't think of any issues. We'll wait and see what the people who can merge things say.